examples/ffi/tests/rms_norm_test.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2024 The JAX Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from absl.testing import absltest
+
+import jax
+import jax.numpy as jnp
+from jax._src import test_util as jtu
+
+try:
+  from jax_ffi_example import rms_norm
+except ModuleNotFoundError:
+  raise unittest.SkipTest("FFI example project was not built")
```

Comment:
Would it be better to explicitly exclude this test from running when we expect it to fail? I worry that a problematic build change in the CI would go under the radar because the test would be skipped.

Reply:
Good point! I didn't initially do that because I wanted to make sure that a user running `pytest` without any flags wouldn't end up trying to collect this test, but I _think_ that adding an `--ignore` to `tool.pytest.ini_options.addopts` in `pyproject.toml` should do the trick?

Reply:
Yeah, sounds promising. But wouldn't that ignore statement also apply in the CI job in this PR? Can we un-ignore a file from the command line?

Reply:
Yes - it turns out that we can! I tested locally and you can see in the GitHub Actions logs [here](https://github.com/google/jax/actions/runs/10708879370/job/29692211149?pr=23409#step:9:18) that the relevant tests ran even after this was added to the global config.

From my experiments, it looks like running:

```bash
pytest --ignore=some/path/to/test some/path/to/test
```

causes tests to be discovered in `some/path/to/test` (`examples/ffi` in this case) even though it was ignored using `--ignore`.

Reply:
So `--ignore` really means `--toggle-ignore-state`?

Reply:
Ah, no! What I meant was a specific path passed at the command line overrides `--ignore`. Sorry that I wasn't clear.

In this case I'm executing:

```bash
pytest exampes/ffi
```

and `--ignore=examples/ffi` is included in the global config.